<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraticulesDialog</class>
 <widget class="QDialog" name="ConfigureGraticulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Graticules</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qt-resources/qt_widgets.qrc">
    <normaloff>:/GPlates-icon-160x160-opaque.png</normaloff>:/GPlates-icon-160x160-opaque.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="graticule_properties_groupbox">
     <property name="title">
      <string>Graticule Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="latitude_delta_label">
        <property name="text">
         <string>La&amp;titude delta:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>latitude_delta_spinbox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="latitude_delta_spinbox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="longitude_delta_label">
        <property name="text">
         <string>Lo&amp;ngitude delta:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>longitude_delta_spinbox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="longitude_delta_spinbox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="colour_label">
        <property name="text">
         <string>Co&amp;lour:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="colour_button_placeholder_widget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>203</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="line_width_hint_spinbox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="line_width_hint_label">
        <property name="text">
         <string>Line width hint:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="main_buttonbox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>latitude_delta_spinbox</tabstop>
  <tabstop>longitude_delta_spinbox</tabstop>
 </tabstops>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc"/>
 </resources>
 <connections/>
</ui>
